tests/providers/google/marketing_platform/operators/test_display_video.py

```diff
@@ -414,15 +415,23 @@ def test_execute(self, mock_temp, gcs_mock_hook, mock_hook):
 
 
 class TestGoogleDisplayVideo360CreateSDFDownloadTaskOperator(TestCase):
+    @mock.patch(
+        "airflow.providers.google.marketing_platform.operators."
+        "display_video.GoogleDisplayVideo360CreateSDFDownloadTaskOperator.xcom_push"
+    )
     @mock.patch(
         "airflow.providers.google.marketing_platform.operators.display_video.GoogleDisplayVideo360Hook"
     )
-    def test_execute(self, mock_hook):
+    def test_execute(self, mock_hook, xcom_mock):
         body_request = {
             "version": "1",
             "id": "id",
             "filter": {"id": []},
         }
+        test_name = 'test_task'
+        mock_hook.return_value.create_sdf_download_operation.return_value = {
+            "name": test_name
+        }
```

Comment:
```suggestion
        mock_hook.return_value.create_sdf_download_operation.return_value = {"name": test_name}
```

I think this will make the static checks happy